rust/lance-encoding/src/encoder.rs

```diff
@@ -173,6 +186,12 @@ impl Default for EncodedColumn {
 /// A task to create a page of data
 pub type EncodeTask = BoxFuture<'static, Result<EncodedPage>>;
 
+/// A buffer of encoded metadata to be placed in the column metadata
+pub struct EncodedMetadataBuffer {
+    // Different parts that will be written to a single buffer on disk
+    pub parts: Vec<Buffer>,
+}
+
```

Comment:
Is this needed / used?

Reply:
No sorry, this is vestigial. Will remove

Reply:
Removed this